app/controllers/application_controller.rb

```diff
@@ -1,23 +1,24 @@
 class ApplicationController < ActionController::Base
-around_action :switch_locale
+ before_action :switch_locale
```

Comment:
indent = 2 spaces
Fix the same error in this file
```suggestion
  before_action :switch_locale
```